pyproject.toml

```
@@ -10,7 +10,7 @@ repository = "https://github.com/jmorganca/ollama-python"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-httpx = "^0.25.2"
+httpx = "<1"
```

```suggestion
httpx = ">=0.25.2,<1"
```

Thoughts on this to preserve the current compatibility?

---

That should work fine since Home Assistant is on `httpx==0.27.0` :+1: